core/new-gui/src/app/common/service/user/auth.service.ts

```diff
@@ -63,18 +63,28 @@ export class AuthService {
    */
   public googleAuth(): Observable<Readonly<{ accessToken: string }>> {
     return this.googleAuthService.getAuth().pipe(
-      mergeMap((auth: GoogleAuth) =>
-        // grantOfflineAccess allows application to access specified scopes offline
-        from(auth.grantOfflineAccess()).pipe(
-          mergeMap(({ code }) =>
-            this.http.post<Readonly<{ accessToken: string }>>(
-              `${AppSettings.getApiEndpoint()}/${AuthService.GOOGLE_LOGIN_ENDPOINT}`,
-              { authCode: code }
-            )
-          )
+      mergeMap(auth => from(auth.grantOfflineAccess())),
+      mergeMap(res =>
```

Comment:
I still suggest we use `{ code }` rather than `res` and then `res.code`, because we are intentionally not using other fields from the res.